src/test_reconciler.js

```diff
@@ -13,6 +13,10 @@ import type {TestFileAssertionStatus, TestAssertionStatus, TestReconciliationSta
 
 import type {FormattedAssertionResult, FormattedTestResults} from '../types/TestResult';
 
+// Regex which defines which short messages are considered to be valid
+// \s\S - stands for any character including new line https://stackoverflow.com/questions/1068280/javascript-regex-multiline-flag-doesnt-work
+const validShortMessage = /expected[\s\S]*received/i;
```

Comment:
If we stick to this, it's maybe worth adding [@babel/plugin-transform-dotall-regex](https://babeljs.io/docs/en/babel-plugin-transform-dotall-regex) and just use the [dotAll flag](https://github.com/tc39/proposal-regexp-dotall-flag) (/s).

Reply:
Are you sure about adding another plugin just for this regex? I would stay with the current approach as it is working well without adding new dependencies.

Reply:
I see it less as adding another plugin but as using ES2018 in order to simplify the syntax. But that is just my personal point of view.

Reply:
That's true, the syntax is indeed a bit simpler with dotAll flag. I've added this improvement as well as removed case-insensitivity according to the previous suggestion.

Reply:
👍
> The only improvement I can make would be /Expected .*[r|R]eceived/s but I don't think it is an improvement worth adding.

That was true, but I'm fine with both version.